### DISCUSSION TITLE
A Question About utils.utils.py

### DISCUSSION THREAD

**Discussion by BugMaker2002** [Dylan-H-Wang/SLF-RPM#10] [Q&A] [answered]
In utils.utils.py, why do we start at 306 frames to 2135 frames in the video? Do these two numbers have any special significance?
![image](https://github.com/Dylan-H-Wang/SLF-RPM/assets/110211332/17709fdc-ad59-4e18-935b-80f2e197a426)

**Comment by Dylan-H-Wang** [comment, marked as answer]
You can choose any numbers you like.
This setting is following a previous paper [DeepPhys](https://openaccess.thecvf.com/content_ECCV_2018/papers/Weixuan_Chen_DeepPhys_Video-Based_Physiological_ECCV_2018_paper.pdf)
